internal/k8s/status.go

```diff
@@ -560,6 +652,21 @@ func (su *statusUpdater) generateExternalEndpointsFromStatus(status []api_v1.Loa
 	return externalEndpoints
 }
 
+func (su *statusUpdater) generateExternalListenerEndpointsFromStatus(status []api_v1.LoadBalancerIngress) []conf_v1alpha1.ExternalEndpoint {
+	var externalEndpoints []conf_v1alpha1.ExternalEndpoint
+	for _, lb := range status {
+		port := su.externalServicePorts
+		if su.bigIPPorts != "" {
+			port = su.bigIPPorts
```

Comment:
in case of bigIPPorts, `u.bigIPPorts` will have `[80,443]` here. We need to choose just `443` (assuming we only report IP for Passthrough TransportServers)